devices/common_patches/h5.yaml

```
@@ -1 +1,78 @@
-_clear_fields: ["*"]
\ No newline at end of file
+_clear_fields: ["*"]
+
+RCC:
+  _strip:
+    - "RCC_"
+
+ADC,ADC?:
+  _strip:
+    - "ADC_"
+
+DBGMCU:
+  _strip:
+    - "DBGMCU_"
+
+GPDMA1:
+  _strip:
+    - "GPDMA_"
+
+CRC:
+  _strip:
+    - "CRC_"
+
+RTC:
+  _strip:
+    - "RTC_"
+
+EXTI:
+  _strip:
+    - "EXTI_"
+  _modify:
+    _interrupts:
+      EXTI4_5:
+        name: EXTI4_15
+        description: "EXTI line 4 to 15 interrupt"
+
+GPIO?:
+  _strip:
+    - "GPIO?_"
+
+USART?:
+  _strip:
+    - "USART_"
+
+SPI?:
+  _strip:
+    - "SPI_"
+
+I2C?:
+  _strip:
+    - "I2C_"
+
+IWDG:
+  _strip:
+    - "IWDG_"
+
+PWR:
+  _strip:
+    - "PWR_"
+
+WWDG:
+  _strip:
+    - "WWDG_"
+
+TIM[13]:
+  _strip:
+    - "TIM?_"
+
+TIM3:
+  _delete:
+    - CNT
+  _modify:
+    CNT_alternate:
+      name: "CNT"
+
+TIM*:
+  _strip:
+    - "TIM*__"
+    - "TIM?*_"
```

I don't know what you wanted to do, but don't use `*` in `_strip` pattern.
`*` means any symbol<b>s</b>

```suggestion
TIM3:
  _delete:
    - CNT
  _modify:
    CNT_alternate:
      name: "CNT"

TIM*:
  _strip:
    - "TIM?_"
    - "TIM??_"
```